src/test/java/org/openrewrite/kotlin/format/TabsAndIndentsTest.java

```diff
@@ -2250,28 +2250,32 @@ fun f(): String? {
             """
               fun f(): String? {
                   val values = (listOf("") as List<String>?)
-                              ?.map { it }
+                      ?.map { it }
```

Comment:
Why do tests like this change if the prefix is unused and unnecessary?
There there code else where like in formatting that references the markers?

Reply:
Ahh, okay, that makes sense. The prefix is added in both positions and printed twice. Thank you!